misc/snapshotter/Dockerfile

```diff
@@ -26,6 +26,14 @@ COPY containerd-nydus-grpc /usr/local/bin/
 RUN chmod +x /usr/local/bin/containerd-nydus-grpc
 COPY nydusd-config.fusedev.json /etc/nydus/config.json
 COPY nydusd-config-localfs.json /etc/nydus/localfs.json
+COPY nydusd-config.fusedev.json /etc/nydus/nydusd-fusedev.json
+COPY nydusd-config-localfs.json /etc/nydus/nydusd-localfs.json
```

Comment:
`nydusd-config-localfs.json` is copied twice (likewise `nydusd-config.fusedev.json`);

Maybe it can be copied to `/etc/nydus/nydusd-localfs.json` only, then create a `/etc/nydus/localfs.json` symlink

Reply:
Thanks.